### DISCUSSION TITLE
cant access the web ui

### DISCUSSION THREAD

**Discussion by farfromrefug** [jonaswinkler/paperless-ng#455] [Support]
It might be a dumb question but i cant seem to find a way to access the webui. Whatever i do i end up on the admin panel.
I installed through docker, redirected port 8000 to 2272.
`http://diskstation.local:2272/` always redirects me to the admin panel.
Same thing if i click on `view site`
Am i missing something?

Thanks

**Comment by jonaswinkler** [comment]
You probably had the old paperless installed before, which installed a permanent redirect to /admin in your browser. Please clear your cache / browsing data to fix that.
